sass-base/_variables.sass

```diff
@@ -0,0 +1,60 @@
+@import 'crucialVariables'
+@import 'functions/index'
+
+// Favorite constants
+$gratio: 1.618 !default
+$pi: 3.14159 !default
+$sqrt2: 1.4142136 !default
+
+// Dimensions
+$spacing: 1.5rem !default
+
+// Font properties
+$font-family-sans-serif: Helvetica, Arial, sans-serif !default
+$font-family-serif: Georgia, Cambria, 'Times New Roman', Times, serif !default
+$font-family-monospace: Monaco, Menlo, Consolas, 'Courier New', monospace !default
+
+$font-family-base: $font-family-serif !default
+$font-family-head: $font-family-sans-serif !default
+
+$fw-thin: 100 !default
+$fw-extraLight: 200 !default
+$fw-light: 300 !default
+$fw-normal: 400 !default
+$fw-medium: 500 !default
+$fw-semiBold: 600 !default
+$fw-bold: 700 !default
+$fw-extraBold: 800 !default
+$fw-black: 900 !default
+
+// Colors
+$c-text: #333333 !default
+$c-text-light: #676767 !default
+$c-text-back: #ffffff !default
+
+// Easings
+$ease-in-cubic: cubic-bezier(0.550, 0.055, 0.675, 0.190) !default
+$ease-out-cubic: cubic-bezier(0.215, 0.610, 0.355, 1.000) !default
+$ease-in-out-cubic: cubic-bezier(0.645, 0.045, 0.355, 1.000) !default
+$ease-in-out-sine: cubic-bezier(0.445, 0.05, 0.55, 0.95) !default
+
+// Breakpoints
+$break320: 20em !default
+$break480: 30em !default
+$break768: 48em !default
+$break992: 62em !default
+$break1200: 75em !default
+$break1296: 81em !default
+$break1400: 87em !default
```

Comment:
Why not use the `em()` helper here?

Reply:
Well, because arguably, it's not going to add any extra degree of readability given the variable names.